src/crimson/os/futurized_store.h

```diff
@@ -24,14 +24,17 @@ class Transaction;
 
 namespace crimson::os {
 class FuturizedCollection;
+class FuturizedShardStore;
 
 class FuturizedStore {
```

Comment:
The division of methods between FuturizedStore and FuturizedShardStore makes sense to me.